### PR TITLE
Add Utkarsh Bansal

### PR DESCRIPTION
**Pull request by Utkarsh3618**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,5 @@ The following students participated in the class:
 - Rahul Jha
 - Jawad
 
+
+- Utkarsh Bansal
```
Comment (RJ722) [inline README.md]: @Utkarsh3618 Nice work. Just one small detail - you added an extra line (line number 10). Can you remove it and then update your PR again?

Comment (Utkarsh3618) [reply README.md]: Done!!
The new commit is below!

**Comment by RJ722** [comment]
Since I've now merged another commit (https://github.com/auvzhcet/git_class_attendance/commit/32781ab50e2af9c2c80b93365f9ecd216f310556) there is another conflict (It happens a lot 😅). You need to update your PR again.

**Comment by Utkarsh3618** [comment]
I left an extra line initially seeing this commit (32781ab) . I will do it again.

**Comment by Utkarsh3618** [comment]
Done!!

**Comment by RJ722** [comment]
It still shows conflicting. In order to update your PR, first pull the changes (which would result in a conflict), then resolve those conflicts. After that commit the new code and push again.

File: README.md [hunk 2]
```diff
@@ -6,3 +6,6 @@ The following students participated in the class:
 
 - Rahul Jha
 - Jawad
+
+
+- Utkarsh Bansal
```
Comment (RJ722) [inline README.md]: @Utkarsh3618 There are multiple empty lines (line no. 9 and 10). Please just remove those and you're good to go.

Comment (RJ722) [inline README.md]: @Utkarsh3618 There are multiple empty lines (line no. 9 and 10). Please just remove those and you're good to go.

**Comment by RJ722** [comment]
You're good to go! \o/

**Comment by Utkarsh3618** [comment]
Thanks!!

**Pull request merged**